x-pack/plugins/observability/public/pages/alerts/components/alerts_flyout/alerts_flyout.tsx

```diff
@@ -45,7 +45,12 @@ export function AlertsFlyout({
   }
 
   return (
-    <EuiFlyout onClose={onClose} size="s" data-test-subj="alertsFlyout">
+    <EuiFlyout
+      className="oblt__alert--flyout oblt__flyout"
```

Comment:
Is `oblt__alert--flyout` used anywhere?

Reply:
No, I will remove it. I started with this and then added more generic `oblt__flyout`, but forgot to remove this.